src/types.js

```diff
@@ -1672,3 +1674,25 @@ export type ReviewRequest = {
   status: ReviewStatus,
   statusChangedAt: string
 };
+
+export type OAuthAuthorizeInput = {
+  clientId: string,
+  redirectUri: string,
+  state: string
+};
+
+export type OAuthOnAuthorizeToken = {
```

Comment:
Since this is the input for getToken

```suggestion
export type OAuthTokenInput = {
```

`OAuthOnAuthorizeToken` seems like what `authorizationCode` is and that's only part of the input